core/src/main/scala/kafka/tools/TestRaftServer.scala

```diff
@@ -238,6 +238,8 @@ class TestRaftServer(
           reader.close()
 
         case Shutdown => // Ignore shutdown command
+
+        case _ => // Ignore other events (such as null)
```

Comment:
I assume that this value is `null` when the timeout is hit. If that is the case then make the case `case null => // Ignore null when timeout expires`.

This should both handle the case when it is null and give us an error if all of the cases/type are not handled.

Reply:
Exactly. When when eventQueue hits the eventTimeoutMs the `null` value arises (as there is nothing to poll from). When debugging encounter that `eventTimeoutMs ` could be 0 (if the throttleTimeMs is equal to 0), so if there is no inmediate events on the eventQueue, the `null` arises inmediatly.

And yes, at the beggining I approached it with a `case null => // Ignore null event`, which made the class run just fine, but let it generic to catch some events that may be being overlooked.

Reply:
Will change on the PR, thanks for the comment!